configure_me/README.md

```diff
@@ -77,13 +92,13 @@ build = "build.rs"
 spec = "config_spec.toml"
 
 [dependencies]
-configure_me = "0.3.3"
+configure_me = "0.3"
 
 [build-dependencies]
-configure_me_codegen = "0.3.11"
+configure_me_codegen = "0.3"
```

Comment:
Damn, just realized I forgot to push the new version - `0.4` to git - it was in crates.io for a while. Pushed it now. Sorry about that.